lib/functions.php

```diff
@@ -174,6 +177,26 @@ function cryptoEnable($socket, array $options = []) {
         }
         if (empty($options["cafile"])) {
             $options["cafile"] = __DIR__ . "/../var/ca-bundle.crt";
+
+            if (class_exists("Phar") && !empty(Phar::running(true))) {
```

Comment:
Either use `Phar::class` or do not `use Phar;` and directly access it via `\Phar`. (I prefer the latter.)